lib/record_store/zone.rb

```diff
@@ -21,19 +21,74 @@ def each(&block)
 
       def find(name)
         return unless File.exists?(zone_path = "#{RecordStore.zones_path}/#{name}.yml")
-        Zone.from_yaml_definition(*YAML.load_file(zone_path).first)
+        load_yml_zone_definition(zone_path).first.last
```

Comment:
We can cleanup this method to be just `defined[name]`, but it's a bit of a yakshave to do without refactoring the stubs in `ZoneTest` (took me a while to dig down). We can do that in a follow-up PR.